src/test/Recordings/ActiveRecordingsTable.test.tsx

```diff
@@ -175,6 +176,8 @@ jest
   .mockReturnValueOnce(of(mockDeleteNotification))
   .mockReturnValue(of()); // all other tests
 
+jest.spyOn(window, 'open').mockReturnValue(null);
```

Comment:
Oh what happened that we need this mocked?

Reply:
The RecordingActions component calls grafanaUpload which calls context.api.grafanaDashboardUrl which wasn't mocked before and for some reason wasn't called either. But now it does, and that has a callback to window.open. In the ArchivedRecordingsTable test there is a similar jest.spyOn so I just reused it.

Reply:
Make sense thanks!!